cli/command/cli_test.go

```diff
@@ -182,18 +182,6 @@ func TestOrchestratorSwitch(t *testing.T) {
 			expectedKubernetes:   false,
 			expectedSwarm:        true,
 		},
-		{
-			doc: "kubernetesIsExperimental",
-			configfile: `{
-				"experimental": "disabled",
-				"orchestrator": "kubernetes"
-			}`,
-			envOrchestrator:      "kubernetes",
-			flagOrchestrator:     "kubernetes",
-			expectedOrchestrator: "swarm",
-			expectedKubernetes:   false,
-			expectedSwarm:        true,
-		},
 		{
 			doc: "kubernetesConfigFile",
 			configfile: `{
```

Comment:
Looks like there's a couple more `"experimental": "enabled"` below; think those can be removed as well?

Reply:
PTAL